source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.js

```diff
@@ -206,6 +237,14 @@ export default class Step1ConfigurationDialog extends Component<Props> {
     }
   );
 
+  isWalletAcceptable = (
```

Comment:
This would be better to sit in the container and passed as a function to the component.

Reply:
Moved